compiler/src/dotty/tools/dotc/transform/patmat/Space.scala

```diff
@@ -894,9 +894,8 @@ object SpaceEngine {
 
 
     if uncovered.nonEmpty then
-      val hasMore = uncovered.lengthCompare(6) > 0
-      val deduped = dedup(uncovered.take(6))
-      report.warning(PatternMatchExhaustivity(showSpaces(deduped), hasMore), m.selector)
+      val deduped = dedup(uncovered)
+      report.warning(PatternMatchExhaustivity(showSpaces(deduped), m), m.selector)
```

Comment:
The `Space` type was probably never meant to be used to render the actual scala code. I wonder if there is a sane way to accomplish that.

Reply:
To be honest I'm not really sure.